pkg/keys/constants.go

```diff
@@ -295,6 +295,7 @@ const (
 	RangeEventTableID = 13
 	UITableID         = 14
 	JobsTableID       = 15
+	SessionsTableID   = 19
```

Comment:
Yeah, 16/17/18 are reserved to reference configuration for non-table entities.

I've updated this section to make it more apparent why this is 19.